lib/primitives.js

```diff
@@ -1,218 +1,258 @@
-var _ = require('lodash');
+var ArrayType, BoolType, ElementType, KeyValueType, NullType, NumberType, ObjectType, StringType, convertFromDom, convertToType, _,
```

Comment:
See https://github.com/apiaryio/drafter.js/blob/master/package.json#L16 and https://github.com/apiaryio/drafter.js/blob/master/scripts/build